boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15-common.dtsi

```diff
@@ -60,7 +60,7 @@
 		sw1 = &button1;
 		sw2 = &button2;
 		sw3 = &button3;
-		watchdog0 = &wdt30;
+		watchdog0 = &wdt31;
```

Comment:
As we change the default watchdog instance here, we need to align the watchdog sample with this change, specifically those files:
https://github.com/zephyrproject-rtos/zephyr/blob/531c457550ca4792b82f21dc1ceb3aaee23bde60/samples/drivers/watchdog/boards/nrf54l15pdk_nrf54l15_cpuapp.overlay
https://github.com/zephyrproject-rtos/zephyr/blob/531c457550ca4792b82f21dc1ceb3aaee23bde60/samples/drivers/watchdog/boards/nrf54l15pdk_nrf54l15_cpuflpr.overlay
https://github.com/zephyrproject-rtos/zephyr/blob/531c457550ca4792b82f21dc1ceb3aaee23bde60/samples/drivers/watchdog/boards/nrf54l15pdk_nrf54l15_cpuflpr_xip.overlay
And test:
https://github.com/zephyrproject-rtos/zephyr/blob/531c457550ca4792b82f21dc1ceb3aaee23bde60/tests/drivers/watchdog/wdt_basic_api/boards/nrf54l15pdk_nrf54l15_cpuapp.overlay

Reply:
done